### PR TITLE
add-scroll-to-omim-table

### PR DESCRIPTION
**Pull request by TereseBo**
This PR adds a functionality by adding a scrollbar to the OMIM-table if more than 700px.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
It's deployed on scout-stage now!

**Comment by dnil** [comment]
Fun! Looks good, and solves the issue. Maybe could be even a little bit smaller on the max size, but the important thing is it doesn't just grow all over the place.
Screenshot 2023-11-07 at 10 16 03

**Review by dnil — APPROVED** [review]
Id say this solves the issue! It tests ok on stage. Size could perhaps be a bit smaller, some common examples are 350, but that is rather arbitrary.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
+## [Unreleased]
```
Comment (northwestwitch) [inline CHANGELOG.md:7]: The changelog item of this PR should be consistent with the other ones. Please add 
`- OMIM table is scrollable if higher than 700px` to the list of items down, from line 14 of this file

**Review by northwestwitch — CHANGES_REQUESTED** [review]
Small fix to the changelog required

**Review by dnil — CHANGES_REQUESTED** [review]
Agree on the changelog - well spotted.

File: CHANGELOG.md (hunk 1 above)
Comment (TereseBo) [reply CHANGELOG.md:7]: I have updated the changelog,I hope I understood the request correctly

**Review by dnil — APPROVED** [review]
👍

**Review by northwestwitch — APPROVED** [review]
Looks good, thanks! The thing is that we don't create a new release every time we merge a PR at least not in this repository. So we can get quite a few changes under the same release

**Pull request merged**